### PR TITLE
added changes to helpers file, and migrations

### PR DESCRIPTION
**Pull request by kevinrczhang**
# Description

Please include a summary of the change and which issue is fixed. Add any screenshots of the change.

Fixes # (ticket #)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: backend/vehicle/helpers.py [hunk 1]
```diff
@@ -69,17 +69,21 @@ def parse_excel_to_vehicle(excel_file):
             "chassis_condition": row["Chassis"],
             "body_condition": row["Body"],
         }
-
+        if len(vehicle_data.get("model_number")) > 10:
+            continue
```
Comment (ltan02) [inline backend/vehicle/helpers.py:73]: Why are we skipping when unicode is more than 10?

Comment (kevinrczhang) [reply backend/vehicle/helpers.py:73]: whoops forgot to remove that line, fixing it rn

Comment (kevinrczhang) [reply backend/vehicle/helpers.py:73]: I'll request review once everything's fixed

File: backend/core/settings.py [hunk 2]
```diff
@@ -13,7 +13,11 @@
 import os
 from datetime import timedelta
 from pathlib import Path
+import environ
 
+# Reads environment variables
+env = environ.Env()
+environ.Env.read_env()
```
Comment (ltan02) [inline backend/core/settings.py:20]: @kevinrczhang We can remove this since we are using `os.environ.get()` now.

**Review by ltan02 — APPROVED** [review]

**Pull request merged**